tests/testthat/_snaps/bayes.md

```diff
@@ -393,12 +393,12 @@
     Message
       x Fold1: preprocessor 1/1:
         Error in `step_spline_b()`:
-        Caused by error in `glue()`:
-        ! Expecting '}'
+        Caused by error in `spline_msg()`:
+        ! Error in if (df < 0)  : missing value where TRUE/FALSE needed
       x Fold2: preprocessor 1/1:
         Error in `step_spline_b()`:
-        Caused by error in `glue()`:
-        ! Expecting '}'
```

Comment:
I'm seeing this change reverted in the snaps on GHA🤔

Reply:
The error we should get is there now; I fixed a bug in recipes that resolved the `glue()` one. Funny: we were grepping against the glue issue in the logging tests here. I converted it to a snapshot.